EDB*Plus: Multi-host connection support update (#4517)
EnterpriseDB/docs: product_docs/docs/edb_plus/41/04_using_edb_plus.mdx

 `connectstring` is the database connection string with the following format:

```text

> <host1>[:<port1>],<host2>[:<port2>],<host3>[:<port3>],..[/<dbname>][?ssl={true | false}][&targetServerType={primary}]

Reviewer: I received clarification from @zahid75 that the other JDBC connection properties and targetServerType values should work but have not been tested with EDB*Plus.  Given that, let's leave the text as is.

define hr_5445="localhost:5445/hr"
```

> Multi-host database connection strings can also be defined in the `login.sq`l file with the `?targetServerType=primary` parameter included in the connection string. The following shows an example how a multi-host connection string can be defined in `login.sql`:

Reviewer 2: ```suggestion
You can also define multi-host database connection strings in the `login.sql` file with the `?targetServerType=primary` parameter included in the connection string. The following shows how you can define a multi-host connection string in `login.sql`:
```
